src/wp-admin/includes/dashboard.php

```diff
@@ -1996,6 +1996,9 @@ function wp_welcome_panel() {
 	?>
 	<div class="welcome-panel-content">
 	<div class="welcome-panel-header">
+		<div class="welcome-panel-header-image">
+			<?php require dirname( __DIR__ ) . '/images/about-header-about.svg'; ?>
```

Comment:
```suggestion
			<?php echo file_get_contents( dirname( __DIR__ ) . '/images/about-header-about.svg' ); ?>
```


If the SVG file contains PHP code, requiring like this will execute the code (at least on some systems).

While that's not a problem on pristine sites, if a site gets hacked somehow, it creates an unexpected location in which the hacker can hide PHP.